### DISCUSSION TITLE
Provide an option to regulate the volume on each app instance individually

### DISCUSSION THREAD

**Discussion by domcote** [File-New-Project/EarTrumpet#1091] [Ideas]
Hey folks - great app. Still one of the first I install on every new Windows.

If an app is open in multiple instances, like multiple Edge windows, ET only lets me change the volume for the whole group simultaneously.

Can you add an option to change the volume per instance/window please?

Use case: streaming audio in one Edge window, watching video in another.

Thanks!

**Comment by virgiliu-prpl** [comment]
This is something I'm also interested in. I'm willing to change the code myself and make a PR but navigating the code is slow since it's undocumented. I would welcome any pointers about which classes and/or methods are relevant. My end goal is to split the applications based on the PID or window title.
